spec/keyboard-commands.spec.js

```diff
@@ -110,5 +110,30 @@ describe('KeyboardCommands TestCase', function () {
             });
             expect(editor.execAction).not.toHaveBeenCalled();
         });
+
+        it('should not execute when the command key are false', function () {
```

Comment:
Actually, you could look at the result of `fireEvent()` and see if `preventDefault()` had been called on it.

That way you could:
```js
var result = fireEvent(... {
    ...
    shiftKey: false
});
expect(result).toBe(true); // true == cancelled

result = fireEvent(... {
    ...
    shiftKey: true
});
expect(result).toBe(false); // false == not cancelled
```

I think that's a better way to check if the functionality does what you expect it to do, what do you think?